share/completions/docker-compose.fish

```diff
@@ -0,0 +1,24 @@
+# Tab completion for docker-compose (https://github.com/docker/compose).
+# Version: 1.9.0
+
+complete -e -c docker-compose
+
+for line in (docker-compose --help | \
+             string match -r '^\s+\w+\s+[^\n]+' | \
+             string trim)
+  set -l doc (echo $line | string split -m 1 ' ')
```

Comment:
Thanks! I changed it in my PR [upstream](https://github.com/docker/compose/pull/4214).